src/lib/pmd/PmdCatalogWrapper.ts

```diff
@@ -129,7 +121,7 @@ export class PmdCatalogWrapper extends PmdSupport {
     });
 
     // Throw all of our events, and then return the paths.
-    this.orderAndEmitEvents(events, []);
+    this.outputProcessor.orderAndEmitEvents(events, []);
```

Comment:
It feels weird for both `orderAndEmitEvents` and `processOutput` to be public methods. I'm not sure how to fix that, but I feel compelled to say it.

Reply:
Agree. Left it this way for now to not expand the scope of the story anymore.